libasn1compiler/asn1c_C.c

```diff
@@ -938,6 +938,7 @@ asn1c_lang_C_type_CHOICE(arg_t *arg) {
 		OUT("_PR_NOTHING,\t/* No components present */\n");
 		TQ_FOR(v, &(expr->members), next) {
 			if(skipComma) skipComma = 0;
+			else if (v->expr_type == A1TC_EXTENSIBLE && !TQ_NEXT(v, next)) OUT("\n");
```

Comment:
Why no comma only if extensible? I would assume that either `v->expr_type == A1TC_EXTENSIBLE` should not be there at all, or it should be inverted into `v->expr_type != A1TC_EXTENSIBLE`. What do you think?